### ISSUE TITLE
Inculde results report instead of the prints

### ISSUE THREAD

**Issue by andrecastro0o** [TIBHannover/ontology2smw#48]
options:
- [x]  -v --verbose all created terms are printed
- [x] -r --report: file to write report
- [x] links to overview pages when writing to wiki in report

Report format/result include

short version, `written` used with --write option 
```
Terms imported:
AEON: 85 terms (written to page Mediaki:Smw_import_aeon)
BFO: 5terms (written to page Mediaki:Smw_import_bfo)
...
```

Verbose

**Comment by andrecastro0o** [comment]
Results in https://github.com/TIBHannover/ontology2smw/commit/3865664cf9b06a39a0454c15fb28fb00d309b323
```
*********** Import Report: ********
obo creates Mediawiki:Smw_import_obo with 28 terms
skos creates Mediawiki:Smw_import_skos with 1 terms
aeon creates Mediawiki:Smw_import_aeon with 154 terms
bfo creates Mediawiki:Smw_import_bfo with 39 terms
```

**Comment by andrecastro0o** [comment]
When writing to wiki (-w) the report includes links to the SMW_import pages

```
*********** Import Report: ***********
obo creates http://192.168.100.100/w/index.php?title=Mediawiki:Smw_import_obo with 28 terms
skos creates http://192.168.100.100/w/index.php?title=Mediawiki:Smw_import_skos with 1 terms
aeon creates http://192.168.100.100/w/index.php?title=Mediawiki:Smw_import_aeon with 154 terms
bfo creates http://192.168.100.100/w/index.php?title=Mediawiki:Smw_import_bfo with 39 terms
```
in https://github.com/TIBHannover/ontology2smw/commit/131b9b579cde3bfc402bdca223c82a9bc0d00d85

**Comment by andrecastro0o** [comment]
closed with #51
